portfolio/src/main/webapp/script.js

```diff
@@ -16,86 +16,66 @@
+    const fact = facts[Math.floor(Math.random() * facts.length)];
 
-  // Add it to the page.
-  const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+    // Add it to the page.
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = fact;
 }
 
 // recieves messages to be presented to the user
 function addMessage() {
-  var numComment = document.getElementById("numInput").value;
-  fetch('/data?numComments=' + numComment).then(response => response.text())
-  .then((message) => {
-  document.getElementById('message-container').innerHTML = message;
-  });
+    var numComment = document.getElementById("numInput").value;
+    fetch('/data?numComments=' + numComment).then(response => response.text())
+    .then((message) => {
+    document.getElementById('message-container').innerHTML = message;
+    });
 }
 
 function deleteData() {
-  const promise = fetch(new Request('/delete-data', {method: 'POST'}));
-  promise.then(() => {
-  addComments();
+    const promise = fetch(new Request('/delete-data', {method: 'POST'}));
+    promise.then(() => {
+    addComments();
     });
 }
 
-// Create the script tag, set the appropriate attributes
-var script = document.createElement('script');
-script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyDjhk-U7NK6nQwmqSScCuZvO3l3_Q88Kps';
-script.defer = true;
-script.async = true;
-
-// Attach your callback function to the `window` object
-window.initMap = function() {
-  // JS API is loaded and available
-};
-
-// Append the 'script' element to 'head'
-document.head.appendChild(script);
-
 function createMap() {
-    document.head.appendChild(script);
   const map = new google.maps.Map(
   document.getElementById('map'),
-  {center: {lat: 35.2271, lng: -80.8431}, zoom: 10});
-
-const trexMarker = new google.maps.Marker({
-    position: {lat: 35.235661, lng: -80.903183},
-    map: map,
-    title: 'My High School'
-  });
-
-const hibachiSpot = new google.maps.Marker({
-    position: {lat: 35.190220, lng: -80.921910},
-    map: map,
-    title: 'This where I had lunch last week'
-  });
-
-  const bossyBeulah = new google.maps.Marker({
-    position: {lat: 35.233580, lng: -80.876500},
-    map: map,
-    title: 'Best place to have chicken sandwiches in Charlotte also conviently 5 mins near me XD'
-  });
-
-    trexMarker.addListener('click', function() {
-          map.setZoom(15);
-          map.setCenter(trexMarker.getPosition());
+    {center: {lat: 35.2271, lng: -80.8431}, zoom: 10});
+
+    var features = [
+      trexMarker = new google.maps.Marker({
+          position: {lat: 35.235661, lng: -80.903183},
+          map: map,
+          title: 'My High School'
+  }),
+
+      hibachiSpot = new google.maps.Marker({
+          position: {lat: 35.190220, lng: -80.921910},
+          map: map,
+          title: 'This where I had lunch last week'
+  }),
+
+      bossyBeulah = new google.maps.Marker({
```

Comment:
since you're using a list, there's no need to do the variable assignment (the bossyBeulah = bit)